test/functional/apps/visualize/_embedding_chart.js

```diff
@@ -51,37 +56,58 @@ export default function ({ getService, getPageObjects }) {
         const data = await PageObjects.visualize.getTableVisData();
         log.debug(data.split('\n'));
         expect(data.trim().split('\n')).to.be.eql([
-          '0B', '2,088',
-          '1.953KB', '2,748',
-          '3.906KB', '2,707',
-          '5.859KB', '2,876',
-          '7.813KB', '2,863',
-          '9.766KB', '147',
-          '11.719KB', '148',
-          '13.672KB', '129',
-          '15.625KB', '161',
-          '17.578KB', '137'
+          '2015-09-20 00:00', '0B', '5',
+          '2015-09-20 00:00', '1.953KB', '5',
+          '2015-09-20 00:00', '3.906KB', '9',
+          '2015-09-20 00:00', '5.859KB', '4',
+          '2015-09-20 00:00', '7.813KB', '14',
+          '2015-09-20 03:00', '0B', '32',
+          '2015-09-20 03:00', '1.953KB', '33',
+          '2015-09-20 03:00', '3.906KB', '45',
+          '2015-09-20 03:00', '5.859KB', '31',
+          '2015-09-20 03:00', '7.813KB', '48'
         ]);
       });
 
       it('should allow to filter in embedded mode', async () => {
-        await filterBar.addFilter('@timestamp', 'is between', '2015-09-19', '2015-09-21');
+        await filterBar.addFilter('@timestamp', 'is between', '2015-09-21', '2015-09-23');
+        await PageObjects.header.waitUntilLoadingHasFinished();
+        await renderable.waitForRender();
+
+        const data = await PageObjects.visualize.getTableVisData();
+        log.debug(data.split('\n'));
+        expect(data.trim().split('\n')).to.be.eql([
```

Comment:
nit: what about trim and split to a const and than log and use it on the expect would

Reply:
I wanted to stay consistent with all our other tests. Wouldn't bother much about that yet, since when we refactor the table vis to React, anyway this method will change completely and instead return a proper 2 level nested array.